podium-gateway/src/main/java/org/bbmri/podium/domain/enumeration/RequestType.java

```diff
@@ -0,0 +1,8 @@
+package org.bbmri.podium.domain.enumeration;
+
+/**
+ * The RequestType enumeration.
+ */
+public enum RequestType {
+    DATA,IMAGES,MATERIAL
```

Comment:
Good point. Ill make it a set in RequestDetail and add a element collection table